src/validation.h

```diff
@@ -676,7 +676,13 @@ class CChainState
     void UnloadBlockIndex() EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
     /** Check whether we are doing an initial block download (synchronizing from disk or network) */
-    bool IsInitialBlockDownload() const;
+    bool IsInitialBlockDownload() const EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
```

Comment:
Would prefer to make `IsIBD` the new semantics

Reply:
Done